### PR TITLE
RIP 7696 : Generic Double Scalar Multiplication (DSM) for any curve

### PR DESCRIPTION
**Pull request by rdubois-crypto**
Double Scalar multiplication is the core operation for non-pairing based ECC protocols (signature, ciphering, key exchange). This RIP proposes to implement a generic double scalar multiplication for any elliptic curve.

### CODE REVIEW

File: RIPS/rip-b4.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
+---
+rip:
```
Comment (abcoathup) [inline RIPS/rip-b4.md:2]:
```suggestion
rip: 7696
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.

File: RIPS/rip-b4.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
+---
+rip: 
+title: Precompile for generic DSM (double scalar multiplication )
```
Comment (abcoathup) [inline RIPS/rip-b4.md:3]:
```suggestion
title: Precompile for generic DSM (double scalar multiplication)
```

**Comment by CarlBeek** [comment]
@rdubois-crypto please add @abcoathup's suggested RIP number to the header, add a eth-magicians link, and fix the table formatting in the constants section then we can get this merged

**Comment by CarlBeek** [comment]
Thanks @rdubois-crypto. There are still some markdown formatting issues (especially the table that is not closed). I made the fixes in https://github.com/rdubois-crypto/RIPs/pull/1, please merge th at (or similar fixes yourself), and I'll merge.

**Review by CarlBeek — APPROVED** [review]

**Pull request merged**